docs/blog/posts/03-deploy-llms-on-a-budget.md

```diff
@@ -0,0 +1,319 @@
+
+## 🤑 What's it going to cost me?
+
+In the example above, we were able to deploy the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model on a single [NVIDIA L4 GPU](https://www.nvidia.com/en-us/data-center/l4/) for ~ $0.22 / hour / replica, or ~$160 / month / replica. This is a significant improvement over the cost of deploying the [mistralai/Mixtral-8x7B-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1) model on 2x [NVIDIA A100-80G GPUs](https://www.nvidia.com/en-us/data-center/a100/), which can cost upwards of $5000 / month (on-demand) on CSPs.
+
+The table below shows the costs of deploying one of these popular MoE LLM models on a single GPU server on GCP. As you can see, the cost of deploying a single model can range from $500 to $5000 / month, depending on the model and ofcourse CSP (kept fixed here). 
+
+
+| Model | Cloud Provider | GPU | Spot | Cost / hr | Cost / month |
+| --- | --- | --- | --- | --- | --- |
+| [mistralai/Mixtral-8x7B-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1) | GCP | NVIDIA A100-80G | - | $7.35 | ~$5290 |
+| [TheBloke/Mixtral-8x7B-Instruct-v0.1-AWQ](https://huggingface.co/TheBloke/Mixtral-8x7B-Instruct-v0.1-AWQ) | GCP | NVIDIA A100-80G | - | $5.03 | ~$3620 |
+| [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) | GCP | NVIDIA L4 | - | $0.70 | ~$500 |
+| **[mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8)** | GCP | NVIDIA L4 | ✅ | **$0.22** | **~$160** |
+
+However, the *onus is on the developer* to figure out the *right instance type, spot instance strategy, and the right number of replicas* to deploy to ensure that the service is both *cost-efficient and performant*. In the coming weeks, we're going to be introducing some exciting tools to help developers alleviate this pain and provide transparency to make the right infrastructure decisions for their services. Stay tuned!
+
+## 🎁 Wrapping up
+
+In this blog post, we showed you how to deploy the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model on a single NVIDIA L4 GPU for under $160 / month and scale-out a dirt-cheap inference service of your own. We used [SkyPilot](https://skypilot.readthedocs.io/) to deploy and manage our [NOS](https://github.com/autonomi-ai/nos) service on spot (pre-emptible) instances, making them especially cost-efficient.
+
+In our next blog post, we’ll take it one step further. We'll explore how you can serve multiple models on the same GPU so that your infrastructure costs **don’t have to** scale with the number of models you serve. The **TL;DR** is that you will soon be able to serve multiple models with fixed and predictable pricing, making model serving more accessible and cost-efficient than ever before.
```

Comment:
Unclear to me if we should be linking blog posts before the fact (even if they are already written) since it requires us to update multiple posts that might already be public.

Reply:
We haven't linked anything yet, but we should be able to modify blogs afterwards too and keep the same urls etc.